### PR TITLE
Fix crash when contact points of a PROTO are requested

### LINKED ISSUE #3516: The `wb_supervisor_node_get_contact_points` crashes a controller in certain cases
**Describe the Bug**
If a Solid (target node) is in a Slot of a PROTO then the `wb_supervisor_node_get_contact_points` function crashes a controller. 

This is due to the changed ID returned by `WbSupervisorUtilities`:
```c++
WbNode* node = WbSupervisorUtilities::getProtoParameterNodeInstance(id, "");
assert(node->uniqueId() == id);  // fails
```
It probably crashes other API functions that rely on the ID.

### PR DESCRIPTION
**Pull request by lukicdarkoo**
Fixes #3516

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
It would be good to add a test for this issue in the test suite in order to prevent regressions.

**Review by stefaniapedrazzi — DISMISSED** [review]
Thank you!

File: tests/api/controllers/center_of_mass_and_contact_points/center_of_mass_and_contact_points.c [hunk 1]
```diff
@@ -27,7 +27,7 @@ int main(int argc, char **argv) {
   ts_assert_boolean_equal(wb_supervisor_field_get_type(children) == WB_MF_NODE, "Children has not the WB_MF_NODE type");
 
   const int n = wb_supervisor_field_get_count(children);
-  ts_assert_boolean_equal(n == 8, "Counting the number of root's children failed (found=%d, expected=8)", n);
+  ts_assert_boolean_equal(n == 9, "Counting the number of root's children failed (found=%d, expected=8)", n);
```
Comment (ad-daniel) [inline tests/api/controllers/center_of_mass_and_contact_points/center_of_mass_and_contact_points.c:30]:
```suggestion
  ts_assert_boolean_equal(n == 9, "Counting the number of root's children failed (found=%d, expected=9)", n);
```

**Review by ad-daniel — COMMENTED** [review]
Minor

File: tests/api/controllers/center_of_mass_and_contact_points/center_of_mass_and_contact_points.c (hunk 1 above)
Comment (lukicdarkoo) [reply tests/api/controllers/center_of_mass_and_contact_points/center_of_mass_and_contact_points.c:30]: Thanks!

**Review by ad-daniel — APPROVED** [review]
Thank you.

**Pull request merged**